introduction.md

```diff
@@ -2,3 +2,4 @@ Adi Pradhan
 Favorite Codeschool Path is Ruby
 * git
 * javascript
+* ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?